website/sidebars.json

```diff
@@ -115,7 +114,6 @@
       "image-style-props",
       "interactionmanager",
       "keyboard",
-      "layout-props",
```

Comment:
@rachelnabors Was `layout-props` page been removed intentionally?

Reply:
That was unintentional! Good catch :)